applications/StructuralMechanicsApplication/custom_utilities/rayleigh_damping_coefficients_utilities.cpp

```diff
@@ -0,0 +1,92 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:             BSD License
+//                                       license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/math_utils.h"
+#include "structural_mechanics_application_variables.h"
+#include "custom_utilities/rayleigh_damping_coefficients_utilities.h"
+
+namespace Kratos
+{
+Vector RayleighDampingCoefficientsUtilities::ComputeDampingCoefficients(Parameters ThisParameters)
+{
+    KRATOS_TRY
+
+    Parameters default_parameters = Parameters(R"(
+    {
+        "echo_level"          : 0,
+        "damping_ratio_0"     : 0.0,
+        "damping_ratio_1"     : -1.0,
+        "eigen_values_vector" : [0.0]
+    })" );
+
+    ThisParameters.RecursivelyValidateAndAssignDefaults(default_parameters);
+
+    // Declaring coefficients
+    Vector coefficients_vector = ZeroVector(2);
+
+    // Getting echo level
+    const int echo_level = ThisParameters["echo_level"].GetInt();
+
+    // The eigen values vector
+    const Vector& r_eigenvalue_vector = ThisParameters["eigen_values_vector"].GetVector();
+
+    // We need at least 2 eigenvalues
+    if (r_eigenvalue_vector.size() > 1) {
+        KRATOS_INFO_IF("ComputeRayleighDampingCoefficientsProcess", echo_level > 0) <<  "The two first eigen values of the structure are:\t" << r_eigenvalue_vector[0] << " Hz\t and " << r_eigenvalue_vector[1] << " Hz" << std::endl;
```

Comment:
```suggestion
        KRATOS_INFO_IF("ComputeRayleighDampingCoefficientsProcess", echo_level > 0) <<  "The two first eigen values of the structure are:\n" << r_eigenvalue_vector[0] << " Hz\t and " << r_eigenvalue_vector[1] << " Hz" << std::endl;
```
otherwise the formatting can get screwed up on small terminals